native-image-workshop/README.md

```diff
@@ -1,51 +1,59 @@
 # GraalVM Native Image Workshop
 
+## Prerequisites
+
+* OS : Linux or Mac OSX
+* Apache Maven
+
 ## Introduction
 
-This workshop aims to walk you through a number of features GraalVM's Native Image offers,
-namely:
+We are going to use a fairly trivial application to walk through using GraalVM Native Image. Along the way we will see:
```

Comment:
>  Along the way you will see: 
Not sure whether we use use 2nd or 3rd person plural.

Reply:
I'll use 3rd, but make sure it is updated throughout